resources/visualizations/racingChart/racing.controllers.js

```diff
@@ -0,0 +1,23 @@
+const { queryByCountryData } = require("./racing.model");
+const DatabaseError = require("../../../server/middleware/DatabaseError");
+
+// Get endpoint Data for the top deaths for Racing Chart Visualization
+const getConfirmedCasesData = async (_req, res, next) => {
+  try {
+    const deathsData = await queryByCountryData();
+    res.status(200).json({
+      deaths: deathsData,
+    });
+  } catch (err) {
+    next(
+      new DatabaseError({
+        message: "Cannot retrieve cases",
+        dbMessage: err,
+      })
+    );
+  }
+};
+
+module.exports = {
+  getConfirmedCasesData,
+};
```

Comment:
Good idea separating this out to a separate folder. Our code base is really modular and easy to navigate this way.